sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/commands.scala

```diff
@@ -47,18 +47,38 @@ case class AnalyzeTable(tableName: String) extends RunnableCommand {
 }
 
 /**
- * Drops a table from the metastore and removes it if it is cached.
+ * Drops a table/view from the metastore and removes it if it is cached.
  */
 private[hive]
 case class DropTable(
```

Comment:
@gatorsmile https://github.com/apache/spark/pull/12121/files#diff-b3f9800839b9b9a1df9da9cbfc01adf8R299 is adding invalidateTable to SessionCatalog.